public/docs/ts/latest/tutorial/toh-pt6.jade

```diff
@@ -396,7 +398,6 @@ block observables-section
     We start by creating `HeroSearchService` that sends search queries to our server's web api.
 
   +makeExample('toh-6/ts/app/hero-search.service.ts', null, 'app/hero-search.service.ts')(format=".")
-
```

Comment:
This newline didn't seem to have any significance, but it messed with syntax highlighting.